datalab/mlalpha/_cloud_models.py

```diff
@@ -51,7 +43,9 @@ def _retrieve_models(self, page_token, page_size):
     page_token = list_info.get('nextPageToken', None)
     return models, page_token
 
-  def __iter__(self):
+  def get_iterator(self):
```

Comment:
Done on Jobs class. The reason of the change is that having a class both exposing methods and exposing itself as iterator is a bit odd and the iterator function is less discoverable.